src/pem/_core.py

```diff
@@ -17,13 +17,19 @@ class AbstractPEMObject(metaclass=ABCMeta):
     """
 
     _pem_bytes: bytes
+    _pem_data: bytes
```

Comment:
I think `data` is a bit too generic. Unless you have a reason against, I would suggest calling it `payload`?